LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/CCF/DataBrokerProtectionUtils.swift

```diff
@@ -22,6 +22,12 @@ import UserScript
 import os.log
 import Combine
 
+func testableAssertionFailure(_ message: @autoclosure () -> String = String(), file: StaticString = #file, line: UInt = #line) {
```

Comment:
IMO this should probably live somewhere more general rather than localized to DBP. Do we do anything like this anywhere else on macOS?